website/content/docs/configuration/worker/pki-worker.mdx

```diff
@@ -22,6 +22,18 @@ worker {
 }
 ```
 
+## KMS Configuration
+PKI Workers credentials can be encrypted by including an optional KMS stanza with the purpose `worker-auth-storage`.
+
+```hcl
```

Comment:
Here and below can you add a comment making it clear that this is an example only and production scenarios should use a different KMS option?

Reply:
Done!